src/server/migrations/20200325162239_crops.js

```diff
@@ -0,0 +1,26 @@
+exports.up = function(knex) {
+  return knex.schema.createTable('crops', function(table) {
+    table.increments();
+    table.varchar('name', 255).notNullable();
+    table.varchar('plant_variety', 255).notNullable();
+    table
+      .integer('fk_user_id')
+      .unsigned()
+      .notNullable();
```

Comment:
You forgot to write the relation here

Reply:
Yes I intentionally did so as I was trying to push and encountered an error that users table cant be found, so as long as we have the user table ready to make a relation, I just did so.

Reply:
Not adding the relation doesn't solve the issue, can you figure out how to resolve or ping the ones you are dependent on @amjadiftikhar